core/src/test/java/org/hisp/dhis/android/core/tracker/importer/TrackerImporterFileResourcesPostCallShould.kt

```diff
@@ -49,16 +51,9 @@ class TrackerImporterFileResourcesPostCallShould {
 
     private lateinit var fileResourcePostCall: TrackerImporterFileResourcesPostCall
 
-    @Before
-    fun setUp() {
-        fileResourcePostCall = TrackerImporterFileResourcesPostCall(
-            fileResourcesPostCall,
-            fileResourceHelper
-        )
-    }
-
+    @OptIn(ExperimentalCoroutinesApi::class)
     @Test
-    fun `Should create a single post call for repeated attributes`() {
+    fun `Should create a single post call for repeated attributes`() = runTest {
```

Comment:
The `runTest` method makes possible to mock suspend functions using the same syntax than before. There is no need to use the `stub` and `onBlocking` methods.